lzy-api/src/main/proto/ai/lzy/v1/workflow/workflow-service.proto

```diff
@@ -67,8 +67,9 @@ message ExecuteGraphResponse {
 // ==================== GraphStatus =====================
 
 message GraphStatusRequest {
-  string executionId = 1;
-  string graphId = 2;
+  string workflow_name = 1;
```

Comment:
after protocol changes we should increment min supported version here https://github.com/lambdazy/lzy/blob/1e291e7b6b4f2bf231525ef384622948f14f64f4/lzy/lzy-service/src/main/java/ai/lzy/service/config/ClientVersions.java#L16